src/kubernetes-ingress-controller/guides/using-gateway-api.md

```diff
@@ -1,6 +1,11 @@
 ---
 title: Using Gateway API
-alpha: true
+
+overrides:
+  alpha:
```

Comment:
Is the feature now in beta, or did it go straight from tech preview/alpha to GA?

Reply:
I removed the changes from Gateway API and will let the KIC team update the file.